packages/data-model/src/models/dictionaries.ts

```diff
@@ -1,6 +1,7 @@
 import { jsonb, pgTable, serial, timestamp, varchar } from 'drizzle-orm/pg-core';
 
-import { SchemaDefinition } from '@overturebio-stack/lectern-client/lib/schema-entities.js';
+import { Schema } from '@overture-stack/lectern-client';
+interface SchemaDefinition extends Schema {}
```

Comment:
Wrapping the `Schema` zod type is needed in order to be used as a column type definition on a Drizzle object.